visualize-admin/visualization-tool
# feat: Tooltip for disabled chart types

Closes #1448

This PR adds tooltip that explains why a given chart type can be selected and consolidates the tooltips used across the editor mode (excluding chart tooltips).

## How to test
1. Go to [this link](https://visualization-tool-git-feat-tooltip-for-disabled-ch-9b046b-ixt1.vercel.app/en/create/new?cube=https://agriculture.ld.admin.ch/foag/cube/MilkDairyProducts/Consumption_Price_Month&dataSource=Prod).
2. ✅ Hover over disabled chart types and see that a tooltip with explanation on why it can't be selected is shown.

@KerstinFaye UI strings can be adjusted via Accent (string ids start with `controls.chart.disabled`), I used the most down-to-code explanations, but maybe they can be tweaked :)
